apps/web/src/modules/create-dao/components/ReviewAndDeploy/ReviewAndDeploy.tsx

```diff
@@ -118,7 +118,7 @@ export const ReviewAndDeploy: React.FC<ReviewAndDeploy> = ({ title }) => {
       sanitizeStringForJSON(setUpArtwork?.projectDescription),
       general?.daoAvatar || '',
       sanitizeStringForJSON(general?.daoWebsite || ''),
-      'https://api.zora.co/renderer/stack-images',
+      'https://nouns.build/api/renderer/stack-images',
```

Comment:
use the const here and `useArtworkPreview`?

Reply:
fixed in [945a22c](https://github.com/BuilderOSS/nouns-builder/pull/497/commits/945a22c0cbc05162ecdd1256334974e4d40f1582)